api/justfile

```diff
@@ -14,13 +14,16 @@ NO_COLOR := "\\033[0m"
 # Version #
 ###########
 
-export API_PY_VERSION := `grep 'python_version' Pipfile | awk -F' = ' '{print $2}' | sed 's/"//g'`
-export PGCLI_VERSION := `grep 'pgcli' Pipfile | awk -F' = ' '{print $2}' | sed 's/\~//g; s/"//g; s/=//g'`
+# These commands use standard GNU tools instead of `pdm` because we do not
+# require PDM on the host machine for API development.
```

Comment:
This statement still holds. If one is working on the API (and even if they are making changes to the attribution package), they do not need PDM on the host (because the package will be mounted into the API container directly and be usable via the API).

PDM is only needed on the host if one is purely working outside the context of the API, like the attribution package directly.